### DISCUSSION TITLE
Videomodul: Content im Hochformat (9:16) sollte verbessert werden, in dem wir eine maximale Höhe oder Letterboxen setzen

### DISCUSSION THREAD

**Discussion by techotline** [lotta-schule/web#233] [Ideen]
### Problem

Videos sollten konstant im Querformat (optimal mit einem Landcape von 16:9) ausgeliefert werden, **damit Beiträge nicht endlos lang sind**, weil der Upload von Movies im Hochformat stattgefunden hat.

Desktop bei Screen: 1440x900; Window: 2880x1640

<img width="1440" alt="Bildschirmfoto 2024-05-04 um 19 15 37" src="https://github.com/lotta-schule/web/assets/153733311/49bfc975-5c3e-46df-8e0c-53ccf9ee4e7f">

### mögliche Lösung

Optional hingegen könnten wir sowohl links als auch rechts einen Letterbox Ratio (das Hinzufügen von "schwarzen Rändern") setzen, die Inhalte horizontal projizieren und das eigentliche Video in der Mitte vertikal platzieren. Ziel ist die Anpassung des Seitenverhältnisses des Videos an das Seitenverhältnis des jeweiligen Bildschirms.

![letterbox_ratio](https://github.com/lotta-schule/web/assets/153733311/245e50d9-2a4d-493a-bcf5-da0ed6ad113b)

![lettrat-image-01](https://github.com/lotta-schule/web/assets/153733311/2c024f16-d666-4a18-a4a1-4f6cb20e03eb)

**Comment by ptitmouton** [comment]
Sehe zwei einfache Möglichkeiten @BillyTheKing 

- Festes Seitenverhältnis von Medien, 1.77 wäre wie von @techotline erwähnt die offensichtlichste Lösung
- Einfach `max-height` setzen. Das erlaubt Flexibilität bei den Formaten mit möglichst wenig schwarz, ginge aber zugunsten der Konsistenz

Sollte beides mit mehr oder weniger einer Zeile css gelöst sein (wollen wir es uns mal schön reden)
